config.toml

```diff
@@ -27,49 +33,79 @@ fractions = false
 date = ["date", ":filename", "publishDate", "lastmod"]
 
 [permalinks]
-  blog = "/:section/:year/:month/:day/:slug/"
+blog = "/:section/:year/:month/:day/:slug/"
 
 # Be explicit about the output formats. We (currently) only want an RSS feed for the home page.
 [outputs]
-home = [ "HTML", "RSS"]
+home = [ "HTML", "RSS", "HEADERS" ]
 page = [ "HTML"]
 section = [ "HTML"]
 
+# Add a "text/netlify" media type for auto-generating the _headers file
+[mediaTypes]
+[mediaTypes."text/netlify"]
+delimiter = ""
+
 [outputFormats]
 [outputFormats.RSS]
 baseName = "feed"
 
+# _headers file output (uses the template at layouts/index.headers)
+[outputFormats.HEADERS]
+mediatype = "text/netlify"
+baseName = "_headers"
+isPlainText = true
+notAlternative = true
+
 [params]
 
 time_format_blog = "Monday, January 02, 2006"
 description = "Production-Grade Container Orchestration"
 showedit = true
 
-latest = "v1.11"
+latest = "v1.12"
 
-fullversion = "v1.11.0"
+fullversion = "v1.11.3"
 version = "v1.11"
-githubbranch = "master"
-docsbranch = "master"
-deprecated = false
+githubbranch = "v1.11.3"
+docsbranch = "release-1.11"
+deprecated = true
 currentUrl = "https://kubernetes.io/docs/home/"
-nextUrl = "http://kubernetes-io-vnext-staging.netlify.com/"
+nextUrl = "https://kubernetes-io-vnext-staging.netlify.com/"
 githubWebsiteRepo = "github.com/kubernetes/website"
 githubWebsiteRaw = "raw.githubusercontent.com/kubernetes/website"
 
+[params.pushAssets]
+css = [
+  "callouts",
+  "styles",
+  "custom-jekyll/tags"
+]
+js = [
+  "custom-jekyll/tags",
+  "script"
+]
+
 [[params.versions]]
```

Comment:
In the past we haven't included links to newer versions in older docs - do we want to commit to keeping this up to date or is the idea that each version at least has a link to the next version?

Reply:
@tfogo In the past, we _have_ updated older versions with newer nav info. 

If you look at the production version of [`config.toml` in `release-1.10`](https://github.com/kubernetes/website/blob/release-1.10/config.toml#L50), you can see that 1.10 was updated when 1.11 released. The updates for 1.12 were missed; that's why this PR and #10668 include updates for 1.12.

As to why we update previous versions: if you visit the [1.11 production build](https://v1-11.docs.kubernetes.io/), you can see that there's no way to navigate back to 1.12 in the version selector. This PR's [preview build](https://deploy-preview-10667--k8sio-v1-11.netlify.com/) shows 1.12 added to the version selector.